types/node-polyglot/index.d.ts

```diff
@@ -22,7 +22,7 @@ declare namespace Polyglot {
         phrases?: any;
         locale?: string;
         allowMissing?: boolean;
-        onMissingKey?: (key: string, options?: Polyglot.InterpolationOptions, locale?: string) => string;
+        onMissingKey?: (key: string, options: Polyglot.InterpolationOptions, locale: string) => string;
```

Comment:
This looks good to me. Strangely, it's actually called with an addition parameter in the source code:

https://github.com/airbnb/polyglot.js/blob/e77ac1a938729bca014b1120cd7fe8f05cc83e90/index.js#L367

However due to it being undocumented, I say this is correct.